lib/companies/pending_changes.ex

```diff
@@ -47,62 +30,66 @@ defmodule Companies.PendingChanges do
     end
   end
 
-  defp apply_changes("create", changeset), do: Repo.insert(changeset)
-  defp apply_changes("update", changeset), do: Repo.update(changeset)
+  def create(changeset, action, user, note \\ "")
+  def create(%{valid?: false} = changes, action, _user, _note), do: invalid_change(changes, action)
+  def create(changes, action, user, note), do: insert_change(changes, action, note, user)
 
-  defp changeset(module, changes) do
-    record =
-      if Map.has_key?(changes, "id") do
-        id = Map.get(changes, "id")
-        Repo.get(module, id)
-      else
-        struct(module, %{})
-      end
+  def act(change_id, "false") do
```

Comment:
I feel like this `act/2` functions is entirely unnecessary. Why can't we pass in a boolean to the existing method?

Reply:
It is passing by the value from the buttons in the form (accept => true, reject => false)